Terminal.Gui/Core/View.cs

```diff
@@ -853,7 +853,7 @@ public override bool OnEnter ()
 		public override bool OnLeave ()
 		{
 			FocusEventArgs args = new FocusEventArgs ();
-			Leave?.Invoke (this, args);
+			Leave?.Invoke (args);
```

Comment:
Just noticed.... why is this logic in `View` and not in the base (`Responder`)? 

Is there an intentional reason for this? It seems more logical to me that since `Responder` defines `OnLeave` (etc...) it should be the one invoking the event. Am I missing something?

Reply:
In `Responder `there is no virtual method that invokes an `Action`.

Reply:
In the `Responder `have this:

```
		public virtual bool OnLeave ()
		{
			return false;
		}

```